<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QCheckBox" name="monitorCheckBox">
      <property name="text">
       <string>Monitor On/Off</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" alignment="Qt::AlignHCenter">
     <widget class="QPushButton" name="recordButton">
      <property name="text">
       <string>Record</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="3">
     <widget class="QGraphicsView" name="imageView"/>
    </item>
    <item row="3" column="0" colspan="3">
     <widget class="QListView" name="savedList"/>
    </item>
    <item row="4" column="0" colspan="3">
     <widget class="QLabel" name="mainStatusLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionCamera_Information"/>
    <addaction name="actionOpen_Camera"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="mainStatusBar"/>
  <action name="actionCamera_Information">
   <property name="text">
    <string>Camera &amp;Information</string>
   </property>
  </action>
  <action name="actionOpen_Camera">
   <property name="text">
    <string>&amp;Open Camera</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
